site/content/get-started.md

```diff
@@ -0,0 +1,311 @@
+```
+
+{{< warning >}}
+The NodePort resource must be deployed in the same namespace as NGINX Gateway Fabric.
+
+If you are making customizations, ensure your `labels:` and `selectors:` also match.
+{{< /warning >}}
+
+---
+
+## Create an example application
+
+In the previous section, you deployed NGINX Gateway Fabric to a local cluster. This section shows you how to deploy a simple web application to test that NGINX Gateway Fabric works.
+
+{{< note >}}
+The YAML code in the following sections can be founded in the [cafe-example folder](https://github.com/nginxinc/nginx-gateway-fabric/tree/main/examples/cafe-example) of the GitHub repository.
+{{< /note >}}
+
+---
+
+### Create the application resources
+
+Create the file _cafe.yaml_ with the following contents:
+
+{{< ghcode "https://raw.githubusercontent.com/nginxinc/nginx-gateway-fabric/refs/heads/main/examples/cafe-example/cafe.yaml">}}
+
+Apply it:
+
+```shell
+kubectl apply -f cafe.yaml
+```
+```text
+deployment.apps/coffee created
+service/coffee created
+deployment.apps/tea created
+service/tea created
+```
+
+Verify that the new pods are in the `default` namespace:
+
+```shell
+kubectl -n default get pods
+```
+```text
+NAME                      READY   STATUS    RESTARTS   AGE
+coffee-6db967495b-dvg5w   1/1     Running   0          80s
+tea-7b7d6c947d-8xmhm      1/1     Running   0          80s
+```
+
+---
+
+### Create Gateway and HTTPRoute resources
+
+Create the file _gateway.yaml_ with the following contents:
+
+{{< ghcode "https://raw.githubusercontent.com/nginxinc/nginx-gateway-fabric/refs/heads/main/examples/cafe-example/gateway.yaml">}}
+
+Apply it using `kubectl`:
+
+```shell
+kubectl apply -f gateway.yaml
+```
+```text
+gateway.gateway.networking.k8s.io/gateway created
+```
+
+Create the file _cafe-routes.yaml_ with the following contents:
+
+{{< ghcode "https://raw.githubusercontent.com/nginxinc/nginx-gateway-fabric/refs/heads/main/examples/cafe-example/cafe-routes.yaml">}}
+
+Apply it using `kubectl`:
+
+```shell
+kubectl apply -f cafe-routes.yaml
+```
+```text
+httproute.gateway.networking.k8s.io/coffee created
+httproute.gateway.networking.k8s.io/tea created
+```
+
+You can check that all of the expected services are available using `kubectl get`:
+
+```shell
+kubectl get service --all-namespaces
+```
+```text
+NAMESPACE       NAME            TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)                      AGE
+default         coffee          ClusterIP   10.96.18.163    <none>        80/TCP                       2m51s
+default         kubernetes      ClusterIP   10.96.0.1       <none>        443/TCP                      4m41s
+default         tea             ClusterIP   10.96.169.132   <none>        80/TCP                       2m51s
+kube-system     kube-dns        ClusterIP   10.96.0.10      <none>        53/UDP,53/TCP,9153/TCP       4m40s
+nginx-gateway   nginx-gateway   NodePort    10.96.186.45    <none>        80:31437/TCP,443:31438/TCP   3m6s
+```
+
```

Comment:
I'd also add that a user should verify that everything is configured properly.

By running `kubectl describe httproutes ...` and `kubectl describe gateways ...` on the resources that were created should show successful conditions in the `status` section